### PR TITLE
Improve find lesson

### PR DESCRIPTION
**Pull request by rgaiacs**
Cover

- [x] #178
- [x] #180

### CODE REVIEW

**Comment by ChristinaLK** [comment]
I like this!  only one comment inline.

File: 06-find.md [hunk 1]
```diff
@@ -241,11 +241,38 @@ a `tools` directory that contains the programs `format` and `stats`,
 and an empty subdirectory called `old`.
 
 For our first command,
-let's run `find . -type d`.
+let's run `find .`.
+
+~~~ {.bash}
+$ find . -type d
```
Comment (ChristinaLK) [inline 06-find.md]: remove `-type d`

Comment (gdevenyi) [inline 06-find.md]: This looks fine in the context of the changes, -type f is covered afterwards.

If the types are to be removed, there needs to be a bit more extensive changes.

Comment (gdevenyi) [inline 06-find.md]: Oops, I see the issue. -type d does need to be removed.

**Comment by gdevenyi** [comment]
@rgaiacs can you cleanup that last -type d line, and then we can merge.

**Comment by rgaiacs** [comment]
@gdevenyi Done. Sorry for my long delay.

**Comment by Ry4an** [comment]
LGTM

**Pull request merged**